components/faqs/reverse-selectors.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { withTheme, ThemeProvider } from 'styled-components'
+
+import SectionLayout from '../SectionLayout'
+import LiveEdit from '../LiveEdit'
+
+const scope = { withTheme, ThemeProvider }
```

Comment:
to be deleted? :) seems like you don't need any theming stuff in the LiveEditor here